### PR TITLE
Reformat js snippets and fix typo

### PR DESCRIPTION
**Pull request by brikou**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,13 +66,14 @@ import ReactDOM from 'react-dom';
 import { DragDropContext, Droppable, Draggable } from 'react-beautiful-dnd';
```
Comment (alexreardon) [inline README.md]: Did you base these changes on the projects `eslint` settings?

File: README.md [hunk 2]
```diff
@@ -132,17 +133,14 @@ class App extends Component {
     return (
       <DragDropContext onDragEnd={this.onDragEnd}>
         <Droppable droppableId="droppable">
-          {(provided, snapshot) => (
+          {(provided, snapshot) =>
             <div
               ref={provided.innerRef}
               style={getListStyle(snapshot.isDraggingOver)}
             >
-              {this.state.items.map(item => (
```
Comment (alexreardon) [inline README.md]: when a function inside a render method returns JSX the convention for this codebase is that the returned value is wrapped inside parentheses `( <MyComponent /> )`. I think it helps to read big trees. If you could please update all the examples to go back to using parentheses that would be great

File: README.md [hunk 3]
```diff
@@ -295,8 +291,12 @@ type Props = Hooks & {|
 import { DragDropContext } from 'react-beautiful-dnd';
 
 class App extends React.Component {
-  onDragStart = () => {...}
-  onDragEnd = () => {...}
+  onDragStart = () => {
```
Comment (alexreardon) [inline README.md]: nice

File: README.md [hunk 4]
```diff
@@ -724,15 +695,15 @@ type DragHandleProps = {|
   'aria-grabbed': boolean,
   draggable: boolean,
   onDragStart: () => void,
-  onDrop: () => void
-|}
+  onDrop: () => void,
+|};
```
Comment (alexreardon) [inline README.md]: nice

**Comment by alexreardon** [comment]
Thanks for reaching out to make the docs better @brikou !!! We really appreciate it. Most of these examples where crafted directly in a markdown file so it is good to see them get some love. Did you make these updates by hand or do you have a linting tool that understands code examples in markdown?

**Comment by brikou** [comment]
Hey @alexreardon I've used @dschau/prettier-markdown, it is far from being perfect but does most of the job. If your snippet is not a valid JS then it breaks (fe: a single method not being enclosed in a class, or using "..." without enclosing it in comment).
Do you want me to apply your feedback about enclosing jsx in parenthesis or can we stick with prettier rules ?

**Comment by alexreardon** [comment]
For now for consistency can you please add the parenthesis? I will look at moving the whole codebase over to prettier soon!

**Comment by brikou** [comment]
@alexreardon done :)

**Comment by alexreardon** [comment]
Thanks heaps @brikou !!! 

dance

**Comment by brikou** [comment]
:smile:

**Pull request merged**